cephci/prereq.py

```diff
@@ -44,6 +63,10 @@
         -k --setup-ssh-keys <BOOL>  Setup SSH keys on cluster
         -f --config <FILE>          CephCI configuration file
         -l --log-level <LOG>        Log level for log utility
+        -p --private-registry <BOOL> Use private registry
```

Comment:
This will change indentation for existing parameters.

Reply:
Indentation fixed for all